mappings/net/minecraft/item/ItemFood.mapping

```diff
@@ -1 +1,9 @@
 CLASS none/aek net/minecraft/item/ItemFood
+	FIELD b hunger I
```

Comment:
It's currently hunger, yours suggests an alternative, so :+1: could only possibly mean I'd support your alternative :stuck_out_tongue: